services/hanke-service/src/test/kotlin/fi/hel/haitaton/hanke/logging/DisclosureLogServiceTest.kt

```diff
@@ -367,6 +367,43 @@ internal class DisclosureLogServiceTest {
         verify { auditLogService wasNot Called }
     }
 
+    @Nested
+    inner class SaveDisclosureLogsForDecision {
+        @Test
+        fun `saves log with application details but decision type`() {
+            val application =
+                AlluDataFactory.createApplication(
+                    alluid = 2,
+                    alluStatus = ApplicationStatus.DECISION,
+                    applicationIdentifier = "JS2300050-2"
+                )
+            val expectedObject =
+                """
+                {
+                  "id": ${application.id},
+                  "alluid": ${application.alluid},
+                  "alluStatus": "${application.alluStatus}",
+                  "applicationIdentifier": "${application.applicationIdentifier}",
+                  "applicationType": "${application.applicationType}",
+                  "hankeTunnus": "${application.hankeTunnus}"
+                }"""
+                    .trimIndent()
```

Comment:
Feel free to ignore: would it make sense to trim / create these expected objects in a similar way? The other is in row 69. There could be a helper function (maybe an extension) to do that.

Reply:
I don't see what kind of helper function would help with this.

Reply:
`fun String.toExpected() = trimIndent().replace("\n", "").replace(" ", "")`

Maybe something like that. But the reason why I commented this is that there is more than one way to create the expected log objects in this file and I think it would be beneficial to have only one.